index.html

```diff
@@ -6,93 +6,194 @@
+		<form id="getContact" name="covid19-contact-form" class="card-body">
+			<div class="input-group py-2">
+				<div class="input-group-prepend">
+					<span class="input-group-text material-icons">edit_location</span>
+				</div>
+				<input type="tel" class="form-control" value="110001" pattern="[1-9]{1}[0-9]{5}" onfocus="this.value='';" id="pinCode" aria-label="PinCode" aria-describedby="pinCode" required>
+			</div>
+			<div class="input-group py-2 ">
+				<div class="input-group-prepend ">
+					<span class="input-group-text material-icons">smartphone</span>
+				</div>
+				<input type="tel" id="phone" required class="form-control" value="9999999999"pattern="[1-9]{1}[0-9]{9}" onfocus="this.value='';" aria-label="PhoneNumber" aria-describedby="phoneNumber">
+			</div>
 
-					<div class="input-group card-body">
-						<div class="input-group-prepend">
-							<span class="input-group-text">Pin Code</span>
-						</div>
-						<input type="tel" class="form-control" value="110001" pattern="[1-9]{1}[0-9]{5}" onfocus="this.value='';" id="pinCode" aria-label="PinCode" aria-describedby="pinCode" required>
-					</div>
-					<div class="input-group card-body">
-						<div class="input-group-prepend ">
-							<span class="input-group-text">PhoneNo</span>
-						</div>
-						<input type="tel" id="phone" required class="form-control" value="9999999999"pattern="[1-9]{1}[0-9]{9}" onfocus="this.value='';" aria-label="PhoneNumber" aria-describedby="phoneNumber">
-					</div>
-					
-					<div class="input-group card-body">
-						<div class="input-group-prepend">
-							<div class="input-group-text">
-								<input type="checkbox" required id="checkboxForWhatsapp" aria-label="Checkbox for subscribing to newsletter on Whatsapp">
-							</div>
-						</div>
-						<label class="form-control" id="phoneNumber">Receive updates on SMS/WhatsApp</label>
-						<div class="input-group-append">
-							<button class="btn btn-block btn-primary" id="susbcribe" type="submit">Subscribe</button>
-						</div>
+			<div class="input-group py-2 ">
+				<div class="input-group-prepend">
+					<div class="input-group-text">
+						<input type="checkbox" required id="checkboxForWhatsapp" aria-label="Checkbox for subscribing to newsletter on Whatsapp">
 					</div>
+				</div>
+				<label class="form-control" id="phoneNumber">Receive updates on SMS</label>
+				<div class="input-group-append">
+					<button class="btn btn-block btn-primary" id="susbcribe" type="submit">Subscribe</button>
+				</div>
+			</div>
+		</form>
+	</main>
+	<aside class="col-md-5 my-4 card">
+		<h2 class="card-header display-4 text-center">About</h2>
+		<p class="card-text muted">
+			We are a group of volunteers with a mission to keep you informed with verified case numbers of Coronavirus in your location, so you can take informed decisions to protect yourself and your community. We came together to work on this project inspired by many others from <a href="coronothon.in">coronothon.in</a>.
```

Comment:
I think we should have https://coronathon.in/ the link and also the text "coronathon.in"

Reply:
Ok, otherwise rest looks ok?